### ISSUE TITLE
.footer/.page-nav color contrast fails WCAG AA

### ISSUE THREAD

**Issue by ghost** [handlebars-lang/docs#51]
I am not colorblind; I am having trouble reading text on `.footer` and `.page-nav`. When using [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/), the color combination for text and background fails tests for WCAG AAA and WCAG AA. The contrast should at least pass WCAG AA for better readability.

**Comment by nknapp** [comment]
Hi and thanks for pointing this out. I hadn't known this tool, but I shall give it a try in the future. 
In this case, I have to admit, I just copied the colors from the original Handlebars site. 

At the moment, I don't have much time to take care of this, but if you like to play around with different colors, please don't hesitate to submit a PR: The changes need to be made in [this directory](https://github.com/handlebars-lang/docs/tree/master/src/.vuepress/styles).

**Comment by nknapp** [comment]
Is there any tool that can be included into the CI/CD-build to check such problems automatically?

**Comment by Afanyiyu** [comment]
There are several tools that can help check the website's "Accessibility" (a11y) such as [stylelint-a11y](https://www.npmjs.com/package/stylelint-a11y), a plugin of [StyleLint](https://stylelint.io/). But I think this does not meet the actual needs.

At present, the tool I found to meet the needs is [Web Accessibility Evaluation Tool (WAVE)](https://wave.webaim.org/), which provides an online inspection page ([here](https://wave.webaim.org/report#/https://handlebarsjs.com/) is an example for Handlebars Docs) and [browser plugins](https://wave.webaim.org/extension/), but it does not provide a free API.

There may be other free tools to solve this problem... but I haven't found one yet.

**Comment by papasmile** [comment]
There are handy tools like https://github.com/nickcolley/jest-axe when we have fully rendered components; but since we are using a site generator I think we'd have to run a full browser-test suite (eg Selenium) to get decent tooling.  That's a significant undertaking even for a small-ish site... perhaps just fix offensive CSS manually for now?

**Comment by nknapp** [comment]
would be cool though to have such a tool for vuepress. It could also be used by other sites...

**Comment by nknapp** [comment]
but I agree, at the moment, it should just be fixed in the CSS. I just always ask myself this question: Could we use tools to prevent this in the future.

**Comment by jaylinski** [comment]
I'd use https://github.com/GoogleChrome/lighthouse-ci for this. It has accessibility checks.

**Comment by jaylinski** [comment]
We are now relying on the [VitePress default-theme](https://vitepress.dev/reference/default-theme-config) to meet accessibility standards.
